cli/src/main.rs

```diff
@@ -46,6 +46,9 @@ enum Opt {
         /// %a with the song album.
         #[structopt(name = "PATTERN", default_value = "%F.%E")]
         output: String,
+        /// Keep input files after transcoding
+        #[structopt(short, long, name = "KEEP")]
```

Comment:
`name` is only used to set the display name of the value taken by options, so it has no effects on flags